codegen.yml

```diff
@@ -0,0 +1,23 @@
+schema: schema.graphql
+documents: 'src/**/*.graphql'
+generates:
+  src/__generated__/types.ts:
+    - typescript
+  src/:
+    preset: near-operation-file
```

Comment:
**1) `near-operation-file` does not remove files if they were removed/renamed.** It knows all files which was created, so it may remove files automatically if fragment/query was renamed/deleted.

Reply:
Yeah you are right :( We decided to keep it this way for now, because otherwise we have to maintain some kind of cache to track all previously generated files. 
But this might change in the future :D